Doc/faq/design.rst

```diff
@@ -599,14 +599,14 @@ Why is there no goto?
 In the 1970s people realized that unrestricted goto could lead
 to messy "spaghetti" code that was hard to understand and revise.
 In a high-level language, it is also unneeded as long as there
-are ways to branch (in Python, with ``if`` statements and ``or``,
-``and``, and ``if-else`` expressions) and loop (with ``while``
-and ``for`` statements, possibly containing ``continue`` and ``break``).
+are ways to branch (in Python, with :keyword:`if` statements and :keyword:`or`,
+:keyword:`and`, and ``if-else`` expressions) and loop (with :keyword:`while`
```

Comment:
`if-else` left as-is since it's not exactly a keyword here

Reply:
You could use `` :keyword:`if-else <if>` `` to make it consistent with the others.